packages/dd-trace/src/plugins/outbound.js

```diff
@@ -52,25 +55,35 @@ class OutboundPlugin extends TracingPlugin {
         }
       }
     }
-    return {}
+    return undefined
   }
 
-  startSpan (name, options) {
```

Comment:
Is the removal of the `startSpan` method intentional?

Reply:
Yes - this is the same as directly calling `TracingPlugin:startSpan`, no point in keeping it.